pyfar/dsp/fft.py

```diff
@@ -240,7 +240,7 @@ def normalization(spec, n_samples, sampling_rate, fft_norm='none',
             norm /= (n_samples * sampling_rate)
         else:
             # Equation 13 in Ahrens et al. 2020
-            norm /= (np.sum(window)**2 * sampling_rate)
+            norm /= (np.sum(window**2) * sampling_rate)
```

Comment:
I think the tests are currently failing because we are passing a window as a list. Changing to 
```suggestion
            norm /= (np.sum(np.asarray(window)**2) * sampling_rate)
```
would be more robust. After that we can see if the tests need to be adapted. Did you check if the inverse normalization works as expected?

Reply:
Indeed, it think the issue relates to the fact that `window` should be of type `np.ndarray`  (I forgot to check this).

Also I did not properly check the inverse normaliation yet.